app/extras/user-status-map.html

```diff
@@ -0,0 +1,30 @@
+<table class="table table-condenced">
+    <thead>
+        <tr>
+            <th>ID</th><th>Name</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>109000100</td><td>Visitor</td>
+        </tr>
+        <tr>
+            <td>109000200</td><td>Active</td>
+        </tr>
+        <tr>
+            <td>109000300</td><td>LoggedIn</td>
+        </tr>
+        <tr>
+            <td>109000400</td><td>Inactive</td>
+        </tr>
+        <tr>
+            <td>109000500</td><td>Banned</td>
+        </tr>
+        <tr>
+            <td>109000600</td><td>Removable</td>
+        </tr>
+        <tr>
+            <td>109000700</td><td>Archived</td>
+        </tr>
+    </tbody>
```

Comment:
of these values, please just keep 109000700, 109000200. the rest are not public facing.

Reply:
Removed all non-public facing statuses. Since each type now has 2 statutes each I combined them on one page.